controllers/memcached/memcached_controller.go

```diff
@@ -181,6 +181,12 @@ func (r *Reconciler) Reconcile(ctx context.Context, req ctrl.Request) (result ct
 	instance.Status.Conditions.Init(&cl)
 	instance.Status.ObservedGeneration = instance.Generation
 
+	if instance.Spec.TLS.MTLS.SslVerifyMode == "Request" || instance.Spec.TLS.MTLS.SslVerifyMode == "Require" {
+		// MTLS cert secrets
+		cl = append(cl, *condition.UnknownCondition(memcachedv1.MTLSInputReadyCondition, condition.InitReason, condition.InputReadyInitMessage))
+		instance.Status.MTLSCert = ""
+	}
```

Comment:
we should set it to `instance.Status.MTLSCert = ""` in an else for switching to `None`

Reply:
shall I extend this loop with the else or do we want to use the one validating the certificate?

Reply:
I think its ok to just add the else here